lib/quby/answers/services/updates_answers.rb

```diff
@@ -13,6 +13,7 @@ def initialize(answer)
 
         def update(new_attributes = {})
           answer.raw_params = new_attributes
```

Comment:
Voorstel is om je clone en rescue naar een losse functie te gooien die een block meekrijgt wat de originele funcionaliteit bevat. Hierdoor krijg je niet een enorme functie waarin je bovenin dingen zit te doen om onderin weer dingen mee te kunnen doen.